tests/mentorship_relation/test_dao_listing.py

```diff
@@ -38,16 +38,72 @@ def setUp(self):
         db.session.add(self.mentorship_relation)
         db.session.commit()
 
+        self.accepted_mentorship_relation = MentorshipRelationModel(
```

Comment:
This code can be a bit organized, initially declare all the dictionaries required. Then add them all and commit the changes. That would produce the similar behavior with a bit clean code right?

Reply:
Right, don't know why but I thought that it's required to call `commit` after every `add`